tests/models/musicgen/test_modeling_musicgen.py

```diff
@@ -801,6 +802,7 @@ def prepare_config_and_inputs_for_common(self):
 @require_torch
 class MusicgenTest(ModelTesterMixin, GenerationTesterMixin, PipelineTesterMixin, unittest.TestCase):
     all_model_classes = (MusicgenForConditionalGeneration,) if is_torch_available() else ()
+    all_generative_model_classes = ()  # See `greedy_sample_model_classes` below
```

Comment:
when I see this comment, I don't really see very clearly the reason to set it to `()` here

Reply:
I'll rewrite to clarify these comments :)

(in essence, this overwrites the original `all_generative_model_classes` such that no model runs the tests in `GenerationTesterMixin `)